### ISSUE TITLE
Screen capture does not work in 0.13 rc3

### ISSUE THREAD

**Issue by franchuk** [nwjs/nw.js#4579]
After selecting the screen or window i always get this:
 failure NavigatorUserMediaError {name: "InvalidStateError", message: "", constraintName: ""}

```
nw.Screen.chooseDesktopMedia(
        ["window","screen"], function(streamId) {
             var vid_constraint = {
                  mandatory: {
                    chromeMediaSource: 'desktop', 
                    chromeMediaSourceId: streamId, 
                    maxWidth: 1920, 
                    maxHeight: 1080,
                    minFrameRate: 1,
                    maxFrameRate: 5
                  }, 
                  optional:[]
                };
                navigator.webkitGetUserMedia({audio:false,video: vid_constraint},
                    function(stream){
                        document.getElementById('video_1').src = URL.createObjectURL(stream);
                        stream.onended = function() { console.log("Ended"); };
                    }, 
                    function(error){
                        console.log('failure',error); 
                      // failure NavigatorUserMediaError {name: "InvalidStateError", message: "", constraintName: ""}
                });
    });

```

**Comment by rogerwang** [comment]
Is it duplicate of #4459 ?

**Comment by ghostoy** [comment]
I can't reproduce this issue. `nw.Screen.chooseDesktopMedia` is literally the same as `chrome.desktopCapture.chooseDesktopMedia`. I tested on Windows, Mac and Linux (Ubuntu).

@franchuk  Could you provide a full sample app?

**Comment by franchuk** [comment]
Here is my app
[app.nw.zip](https://github.com/nwjs/nw.js/files/177602/app.nw.zip)

**Comment by ghostoy** [comment]
This only happens on node-remote page. I'll look into it.

**Comment by ghostoy** [comment]
@rogerwang There is similar issue with `nw.Screen.DesktopCaptureMonitor`. I've submitted another PR to fix it.

**Comment by ghostoy** [comment]
This issue was fixed since 0.13.0-rc4.
